README.md

```diff
@@ -5,18 +5,23 @@ Home for discussion about Sous Chefs, and various guides for cookbook authors.
 Live discussion happens on [![Chef Community Slack](https://community-slack.chef.io/badge.svg)](https://community-slack.chef.io/) in [#sous-chefs](https://chefcommunity.slack.com/messages/sous-chefs/).
 
 Cookbooks:
-- [`bsdcpio`](https://github.com/sous-chefs/bsdcpio_cookbook)
-- [`dhcp`](https://github.com/sous-chefs/dhcp-cook)
-- [`etcd`](https://github.com/sous-chefs/etcd-cookbook)
-- [`mongodb`](https://github.com/sous-chefs/mongodb-cookbook)
-- [`nano`](https://github.com/sous-chefs/nano-cookbook)
-- [`unbound`](https://github.com/sous-chefs/unbound-cookbook)
-
+- [`apache2`](https://github.com/sous-chefs/apache2)
+- [`bsdcpio`](https://github.com/sous-chefs/bsdcpio)
+- [`chef-compliance`](https://github.com/sous-chefs/chef-compliance)
+- [`dhcp`](https://github.com/sous-chefs/dhcp)
+- [`etcd`](https://github.com/sous-chefs/etcd)
+- [`graphite`](https://github.com/sous-chefs/graphite)
+- [`mongodb`](https://github.com/sous-chefs/mongodb)
+- [`nano`](https://github.com/sous-chefs/nano)
+- [`postgresql`](https://github.com/sous-chefs/postgresql)
+- [`unbound`](https://github.com/sous-chefs/unbound)
+- [`sysctl`](https://github.com/sous-chefs/sysctl)
```

Comment:
[`haproxy`](https://github.com/sous-chefs/haproxy) seems to be missing in the list.